src/sections/Meshery/Features-Col/index.js

```diff
@@ -55,7 +51,7 @@ const Features = () => {
                       duration={6}
                       separator=","
                       end={feature.count.value !== 0 ? feature.count.value : performanceCount }
-                      suffix={feature.count.description == "service meshes supported" ? "" : "+"} />
+                      suffix={feature.count.description == "users" ? "+" : ""} />
```

Comment:
Did we have the wrong description before?

Reply:
No, just restricted the use of `+` symbol to `users` count. While others don't have it.

Reply:
Oh. 😃